.travis.yml

```diff
@@ -29,6 +29,7 @@ env:
     - PA_CLIENT_ID=fakeclientid.apps.googleusercontent.com
     - PA_CLIENT_SECRET=fakeclientsecret
     - PARTICLE_WEBHOOK_API_KEY_TEST=webhookKey
+    - CONSECUTIVE_OPEN_DOOR_HEARTBEAT_THRESHOLD_TEST=432
```

Comment:
Good that you added this here! (Is 432 the number of heartbeats in 5 days?)

Reply:
432 is an arbitrary number of days in heartbeats (not 5). Was only using it for test cases, but we should definitely decide on the actual number of days on when to send the alert.

Reply:
Kewl